src/metabase/util/grouper.clj

```diff
@@ -7,18 +7,33 @@
   - Suitable for use cases that can tolerate lag time in processing. For example, updating
     last_used_at of cards after a query execution. Things like recording view_log should not use
     grouper since it's important to have the data immediately available.
-  "
+
+
+  Batch processing can be disabled by setting the environment variable `MB_DISABLE_GROUPER_BATCH_PROCESSING=true`."
   (:require
    #_{:clj-kondo/ignore [:discouraged-namespace]}
    [grouper.core :as grouper]
+   [metabase.models.setting :refer [defsetting]]
+   [metabase.util.i18n :refer [deferred-tru]]
    [potemkin :as p])
   (:import
    (grouper.core Grouper)))
 
+(set! *warn-on-reflection* true)
+
 (comment
  p/keep-me
  Grouper/keep-me)
 
+(defsetting disable-grouper-batch-processing
```

Comment:
The feature flag we want for the backfill disables the updates completely, to avoid the load. This setting may have the opposite effect, so I'm  bit skeptical of introducing it.

How about rather having a setting that skips the submit?

Reply:
The only reason I introduced this setting is because we have an e2e test that tries to assert view count, so I need a way to make all the batched tasks process synchronously. Disabling it would not solve that problem.

Reply:
Ah, gotcha